Dockerfile

```diff
@@ -1,10 +1,9 @@
-FROM gliderlabs/alpine:3.3
+FROM gliderlabs/alpine:3.4
 MAINTAINER Hypothes.is Project and contributors
 
 # Install system build and runtime dependencies.
-RUN apk add --no-cache ca-certificates python3 nodejs curl \
-  && curl -sSL https://bootstrap.pypa.io/get-pip.py | python3 \
-  && rm -rf /var/cache/apk/*
+RUN apk-install ca-certificates curl nodejs python3 \
+  && curl -sSL https://bootstrap.pypa.io/get-pip.py | python3
```

Comment:
It looks like Alpine fixed their Python 3 package in 3.4, so it might be worth applying this diff:

```diff
---
 Dockerfile | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/Dockerfile b/Dockerfile
index 74a8b20..a8f7daa 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -2,8 +2,7 @@ FROM gliderlabs/alpine:3.4
 MAINTAINER Hypothes.is Project and contributors

 # Install system build and runtime dependencies.
-RUN apk-install ca-certificates curl nodejs python3 \
-  && curl -sSL https://bootstrap.pypa.io/get-pip.py | python3
+RUN apk-install ca-certificates curl nodejs python3

 # Create the bouncer user, group, home directory and package directory.
 RUN addgroup -S bouncer \
@@ -16,7 +15,7 @@ COPY README.rst package.json requirements.txt ./
 RUN npm install --production \
   && npm cache clean

-RUN pip install --no-cache-dir -r requirements.txt
+RUN pip3 install --no-cache-dir -U pip && pip3 install --no-cache-dir -r requirements.txt

 COPY . .

--
```